core/src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -649,14 +663,14 @@ class CassandraReadJournal protected (
               c.prepareSelectHighestNr,
               c.preparedSelectDeletedTo,
               s,
-              querySettings.readProfile),
+              readProfile),
             settings,
+            executionContext,
             fastForwardEnabled))
         .named(name)
     }.mapAsync(querySettings.deserializationParallelism) { row =>
-        extractor.extract(row, deserializeEventAsync)
-      }
-      .withAttributes(ActorAttributes.dispatcher(querySettings.pluginDispatcher))
+      extractor.extract(row, deserializeEventAsync)
+    }
```

Comment:
Dispatcher here removed because done everywhere this is called instead, right?

Reply:
That is right. The intention is that we shouldn't introduce an async boundary here.
It's typically used like this:
```
          queries.eventsByPersistenceId(...)
          .map(...)
          .toMat(Sink.ignore)(Keep.right)
          .withAttributes(ActorAttributes.dispatcher(settings.journalSettings.pluginDispatcher))
          .run()
```

I see now that we have a slightly different usage with `mapAsync:
```
          queries.eventsByPersistenceId(...)
          .mapAsync(...)
          .toMat(Sink.ignore)(Keep.right)
          .withAttributes(ActorAttributes.dispatcher(settings.journalSettings.pluginDispatcher))
          .run()
```

For that case I wonder if the dispatcher propagates all the way back to the `eventsByPersistenceId` since there is already an async boundary via the `mapAsync`?